test/test.js

```diff
@@ -822,5 +822,41 @@ exports.shuffleArray = shuffleArray_1;
 `
 			);
 		});
+
+		it.only('normalizes paths used in the named export map', async () => {
```

Comment:
Probably not a good idea to check in an `.only` 😉

Reply:
Pff, details.